src/ReusableFunctions/funcs.jsx

```diff
@@ -0,0 +1,35 @@
+export function searchLikes(state, course) {
+    if (state.likedVideo.filter((item) => item.id === course.id).length === 0) {
```

Comment:
You can directly return the condition inside the `if` block, everything else is redundant

Reply:
Sure, will refactor this.